src/deployment/fabricProvisioningHelpers.ts

```diff
@@ -65,7 +66,7 @@ export async function initializeFabricProvisioningState(
 
     state.formState = {
         accountId: defaultAccountId,
-        groupId: groupOptions[0].value,
+        groupId: selectedGroupId || groupOptions[0]?.value || "",

```

Comment:
What case does the extra `|| ""` cover?  groupOptions should always have at least one (the root group)